src/Catalyst.Core.Modules.KeySigner/KeySigner.cs

```diff
@@ -40,6 +42,7 @@ public sealed class KeySigner : IKeySigner
         private readonly ICryptoContext _cryptoContext;
         private readonly IKeyRegistry _keyRegistry;
         private readonly KeyRegistryTypes _defaultKey = KeyRegistryTypes.DefaultKey;
+        static readonly ArrayPool<byte> Pool = ArrayPool<byte>.Shared;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'Pool'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4569935)